paper.md

```diff
@@ -0,0 +1,98 @@
+---
+title: 'HOHQMesh: An all Quadrilateral/Hexahedral Unstructured Mesh Generator for High Order Elements'
+tags:
+  - Mesh Generation
+  - Spectral Element Methods
+  - High Order
+  - Unstructured Quadrilateral and Hexahedral Meshes
+authors:
+  - name: David A. Kopriva
+    orcid: 0000-0002-8076-0856
+    affiliation: "1, 2"
+  - name: Andrew R. Winters
+    orcid: 0000-0002-5902-1522
+    corresponding: true
+    affiliation: 3
+  - name: Michael Schlottke-Lakemper
+    orcid: 0000-0002-3195-2536
+    affiliation: 4
+affiliations:
+ - name: Professor Emeritus, The Florida State University, Tallahassee, FL, USA
+   index: 1
+ - name: Adjunct Professor, San Diego State University, San Diego, CA, USA
+   index: 2
+ - name: Department of Mathematics; Applied Mathematics, Linköping University, Sweden
+   index: 3
+ - name: Professor, High-Performance Scientific Computing & Centre for Advanced Analytics and Predictive Sciences, University of Augsburg, Germany
+   index: 4
+date: 01 November 2024
+bibliography: paper.bib
+---
+
+
+# Summary
+
+HOHQMesh generates unstructured all-quadrilateral and hexahedral meshes with high order boundary information for use with spectral element solvers. Model input by the user requires only an optional outer boundary curve plus any number of inner boundary curves that are built as chains of simple geometric entities (lines and circles), user defined equations, and cubic splines. Inner boundary curves can be designated as interface boundaries to force element edges along them. Quadrilateral meshes are generated automatically with the mesh sizes guided by a background grid and the model, without additional input by the user. Hexahedral meshes are generated by extrusions of a quadrilateral mesh, including sweeping along a curve, and can follow bottom topography. The mesh files that HOHQMesh generates include high order polynomial interpolation points of arbitrary order.
+
```

Comment:
```suggestion
HOHQMesh is available on Linux, MacOS, and Windows.

```
Could also be added below under "functionality", but I think it is important to include it somewhere.

Reply:
Since I didn't hear any objections, I'll just add this.